<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>825</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>884</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,13,5,3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="autoAdjust">
              <property name="styleSheet">
               <string notr="true">QRadioButton{   spacing:12px;     color: rgb(0, 0, 0);     background-color: rgba(165, 205, 255,125);     border:1px outset rgb(255, 255, 255);     border-radius:8px;   padding:4px; }

/*鼠标放在按钮上方*/ QRadioButton:hover {    background-color: rgba(165, 205, 255,80%);    border:2px outset rgba(36, 36, 36,0); }
/*鼠标点击按钮*/ QRadioButton:pressed {    background-color: rgba(165, 205, 255,90%);    border:4px outset rgba(36, 36, 36,0); image:url(:/icon/icon/switch_on.svg);}

/*已选择按钮*/ QRadioButton:checked {    background-color: rgba(165, 205, 255,90%);    border:4px outset rgba(36, 36, 36,0); color: rgb(255, 16, 143);}

</string>
              </property>
              <property name="text">
               <string>Automatic Adjustment</string>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>4</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Shift+A</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="manualAdjust">
              <property name="styleSheet">
               <string notr="true">QRadioButton{   spacing:12px;     color: rgb(0, 0, 0);     background-color: rgba(165, 205, 255,125);     border:1px outset rgb(255, 255, 255);     border-radius:8px;   padding:4px; }

/*鼠标放在按钮上方*/ QRadioButton:hover {    background-color: rgba(165, 205, 255,80%);    border:2px outset rgba(36, 36, 36,0); }
/*鼠标点击按钮*/ QRadioButton:pressed {    background-color: rgba(165, 205, 255,90%);    border:4px outset rgba(36, 36, 36,0); image:url(:/icon/icon/switch_on.svg);}

/*已选择按钮*/ QRadioButton:checked {    background-color: rgba(165, 205, 255,90%);    border:4px outset rgba(36, 36, 36,0); color: rgb(255, 16, 143);}

</string>
              </property>
              <property name="text">
               <string>Manual Adjustment</string>
              </property>
              <property name="shortcut">
               <string>Shift+M</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="colorTableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>60</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pause">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Pause</string>
              </property>
              <property name="shortcut">
               <string>Shift+P</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clear">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
              <property name="shortcut">
               <string>Shift+C</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="start">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
              <property name="shortcut">
               <string>Shift+S</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="export_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Export</string>
              </property>
              <property name="shortcut">
               <string>Shift+E</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Port</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Baud Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open Serial</string>
                </property>
                <property name="shortcut">
                 <string>Shift+Space</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTipDuration">
                 <number>0</number>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/icon/icon/Refresh.png</normaloff>
                  <normalon>:/icon/icon/Refresh.png</normalon>:/icon/icon/Refresh.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="20,4">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::TabFocus</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="teng">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>TENG</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2"/>
            </widget>
            <widget class="QWidget" name="cap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>Cap</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9"/>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,20">
            <item>
             <widget class="QTableWidget" name="shortcutTableWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="accessibleName">
               <string>shortcutTableWidget</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::Box</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="frameShape">
               <enum>QFrame::Shape::Box</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
